clang/lib/StaticAnalyzer/Checkers/BuiltinFunctionChecker.cpp

```diff
@@ -202,6 +213,11 @@ void BuiltinFunctionChecker::handleOverflowBuiltin(const CallEvent &Call,
   // Calling a builtin with an incorrect argument count produces compiler error.
   assert(Call.getNumArgs() == 3);
 
+  // If ResultType is null, we can't proceed with the evaluation
+  if (ResultType.isNull()) {
+    return;
+  }
```

Comment:
When can `ResultType` be null? Is it that the `getSufficientTypeForOverflowOp` of `__int128` is a null QualType?